src/support.cc

```diff
@@ -1732,7 +1908,8 @@ bool certifier::utilities::ECC_to_key(const EC_KEY* ecc_key, key_message* k) {
   }
 
   k->set_allocated_ecc_key(ek);
-  BN_CTX_free(ctx);
+  if (ctx != nullptr)
+    BN_CTX_free(ctx);
```

Comment:
(Aside: Of all the changes I've seen so far, this one _may_be_ the one causing my sporadic failures.
None of the rest seem relevant as all they do is to add extra printf() calls.)